ConfigurationSystem/Service/ConfigurationHandler.py

```diff
@@ -23,7 +27,7 @@ def export_getCompressedData( self ):
     sData = gServiceInterface.getCompressedConfigurationData()
     return S_OK( sData )
 
-  types_getCompressedDataIfNewer = [ types.StringTypes ]
+  types_getCompressedDataIfNewer = [ StringTypes ]
```

Comment:
Anyway, this type of changes I think can be done in 1 go, bulk, in a future release.